modules/ocf_decal/manifests/website.pp

```diff
@@ -27,38 +27,52 @@
       mode    => '0640';
   }
 
-  apache::vhost { 'decal-redirect':
-    servername => 'decal.ocf.berkeley.edu',
-    serveraliases => [
+  apache::vhost { 'decal-http-redirect':
+    servername      => 'decal.ocf.berkeley.edu',
+    serveraliases   => [
       'decal',
       'decal.ocf.io',
       'decal.xcf.sh',
       'decal.xcf.berkeley.edu'
     ],
+
     port            => 80,
     docroot         => '/srv/www/decal',
     redirect_status => 'permanent',
     redirect_dest   => 'https://decal.ocf.berkeley.edu/',
   }
 
-  apache::vhost { 'decal-ssl':
-    servername => 'decal.ocf.berkeley.edu',
-    serveraliases => [
+  apache::vhost { 'decal-canonical-redirect':
+    servername      => 'decal.ocf.io',
```

Comment:
I feel like there might be a better way to do this, but I can't think of what it is right now. This is fine though, this looks good to me. Thanks for changing this!

Reply:
likewise, but this seemed like the simplest way to implement this